src/PanoImageRenderer/VideoLoader.js

```diff
@@ -36,48 +36,46 @@ export default class VideoLoader {
 
 		sourceElement.src = videoSrc;
 		videoType && (sourceElement.type = videoType);
-		// return sourceElement;
+
 		this._video.appendChild(sourceElement);
+		this._sourceCount++;
```

Comment:
Defining and initializing  _sourceCount property are missing.
It suppose to be in the constructer.

Reply:
@happyhj destroy() function do this. and it is called whenever set() method is called. But as you point out, this function naming is not proper for initializing. it may make confusion while reading code. So I replace it with `_reset`. and `destory` calls `_reset`.